.vscode/settings.json

```diff
@@ -113,7 +113,7 @@
     "bazel-testlogs": true,
     "third_party": false
   },
-  "python.pythonPath": "./python/venv/bin/python",
+  "python.pythonPath": "/home/tin/miniconda3/bin/python",
```

Comment:
revert

Reply:
Done, I didn't notice this setting was changed before